sale_coupon_advanced/models/sale_coupon_program.py

```diff
@@ -72,49 +69,60 @@ def _force_sale_order_lines(self, programs, order):
             ):
                 order.add_reward_line_values(program)
 
+    # FIXME: whats this? Original method does not exist on this model.
```

Comment:
pls move method to sale.order model

Reply:
Why move? This method does not do anything. And there is already implementation on sale.order anyway.

Reply:
it is a notice for me, we left some not finished work and it is defined in sale.order in core

Reply:
Oh, u meant move your TODO comment. OK